src/lib/template-factory.ts

```diff
@@ -89,8 +89,9 @@ export function templateFactory(result: TemplateResult) {
  * joining the strings array.
  */
 export type templateCache = {
-  stringsArray: WeakMap<TemplateStringsArray, Template>;
-  keyString: Map<string, Template>;
+  readonly stringsArray: WeakMap<TemplateStringsArray, Template>; readonly keyString: Map<
```

Comment:
wtf. add a comment at the end of the first line?

Reply:
Huh, yeah. Fixed.